psycopg_c/build_backend/psycopg_build_ext.py

```diff
@@ -56,11 +58,16 @@ def _setup_ext_build(self) -> None:
                     if fext == ".c" and os.path.exists(base + ".pyx"):
                         ext.sources[i] = base + ".pyx"
 
+            
+            compiler_directives = {
+                "always_allow_keywords": False,
+            }
+            if Version(cython_version) >= Version("3.1.0"):
```

Comment:
As min version is now 3.1.1 you can drop this check or replace it with an assert.